.vscode/settings.json

```diff
@@ -2,5 +2,13 @@
   "go.toolsEnvVars": {
     "GOARCH": "wasm",
     "GOOS": "js"
+  },
+  "files.associations": {
```

Comment:
Please remove these files as they are C++ specific but set in the root VSCode configuration